make/build.js

```diff
@@ -121,6 +121,26 @@ module.exports = function build(settings, task) {
     });
 
     //////////////////////////////////////////////////////////////////////
+    // Make sure all examples compile
+    //////////////////////////////////////////////////////////////////////
+
+    task("excomp", [], function() {
+        this.excomp(
+            "examples/**/*.html",
+            "src/js/libcs/Parser.js",
+            function(html, parser) {
+                var re = /<script[^>]*type *= *['"]text\/x-cindyscript['"][^>]*>([^]*?)<\/script>/g;
```

Comment:
@gagern The character group matching inside the script tags seems erroneous. I suppose it should be `[^<]`?